### DISCUSSION TITLE
PR page: incorrect tab width

### DISCUSSION THREAD

**Discussion by Deleplace** [community/community#10136] [Pull Requests]
Reading [this PR](https://github.com/sethvargo/go-retry/pull/11/files) and unfolding the very last chunk of code at the bottom, below l.85 of file retry.go, this is what I see (in Firefox 95.0.1 (64-bit) in Ubuntu):

![image](https://user-images.githubusercontent.com/13508141/149562505-80bc8b23-bc14-4cbd-b789-3836517a1df5.png)

This is unexpected because the two `case`s should have the exact same indentation. This is what the indentation _should_ look like imo (I just shifted some grey LOCs to the left):

![image](https://user-images.githubusercontent.com/13508141/149562651-e8c274f4-abee-4a63-b1af-d6ec86060c91.png)

It is my understanding that the problem comes from some difference between the styles applied to the white background lines, and the styles applied to the grey background lines.

Here I'm selecting (highlighting in blue) a **single** tab char in the white zone:

![image](https://user-images.githubusercontent.com/13508141/149563104-54e3c42e-50b2-4550-928f-d18ef6e8d3f1.png)

And here I'm selecting (highlighting in blue) a **single** tab char in the grey zone:

![image](https://user-images.githubusercontent.com/13508141/149563174-fe240593-03de-4c2a-b92f-6fff3fa129b1.png)

**Comment by Marcono1234** [comment]
I am seeing the same bug for Firefox 98.0.2 (64-Bit) on Windows 10, both with unified and split GitHub pull request view. This also occurs for other pull requests in other projects ([example](https://github.com/junit-pioneer/junit-pioneer/pull/565/files)). I would assume every file using tabs for indentation is affected.

Interestingly I am not experiencing this with Chrome (Version 100.0.4896.60 (64-Bit)) or Edge (Version 99.0.1150.55 (64-bit)).

**Reply by Marcono1234** [reply]
Have reported this to GitHub Support; they said they have received a few reports about this already and are investigating it.
